ethereum/gnosis_safe/view_safes.sql

```diff
@@ -22,6 +22,7 @@ CREATE MATERIALIZED VIEW gnosis_safe.view_safes AS
                 '\x6851d6fdfafd08c0295c392436245e5bc78b0185')  -- mastercopy v1.2.0
             )
         )
+        AND gas_used > 0  -- to ensure the setup call was successful
```

Comment:
@tschubotz i'm unable to drop & recreate the ethereum view to add this filter because another object depends on it: dune_user_generated.blockdatafetch_safecontracts -- assuming this is your object.

Reply:
https://dune.com/queries/1169/1990

> @tschubotz i'm unable to drop & recreate the ethereum view to add this filter because another object depends on it: dune_user_generated.blockdatafetch_safecontracts -- assuming this is your object.

No, unfortunately not my object. 



> also noticed the ethereum view cron job has been inactive since mid-march or so, maybe since it was failing without this filter for gas_used. we can try to enable it after this change gets pushed through (after dependency mentioned in prior comment)

That's odd. It seems like there's still data being added to the view though: https://dune.com/queries/1169/1990

Reply:
okay, since it wasn't your object, i dropped it to be able to rebuild the ethereum view_safes. all objects here should be ready now.
as for the cron job for ethereum, i was looking at the incorrect job. it is running as you found.